src/main/java/org/breedinginsight/brapi/v2/services/BrAPITrialService.java

```diff
@@ -484,11 +501,13 @@ private Map<String, Object> createExportRow(
                 .findFirst();
         blockLevel.ifPresent(brAPIObservationUnitLevelRelationship ->
                 row.put(ExperimentObservation.Columns.BLOCK_NUM, Integer.parseInt(brAPIObservationUnitLevelRelationship.getLevelCode())));
-        if (ou.getObservationUnitPosition() != null && ou.getObservationUnitPosition().getPositionCoordinateX() != null &&
-                ou.getObservationUnitPosition().getPositionCoordinateY() != null) {
+
+        //Row and Column
+        if ( ou.getObservationUnitPosition() != null ) {
```

Comment:
Is it allowed to upload a file that has row but not column data or vice versa? I think this change makes sense regardless.